lib/http-bridge/test/integration/Test/Integration/Framework/DSL.hs

```diff
@@ -438,3 +462,54 @@ wantedErrorButSuccess
     -> m void
 wantedErrorButSuccess =
     fail . ("expected an error but got a successful response: " <>) . show
+
+
+---
+--- CLI
+---
+cardanoWalletLauncherCLI :: CmdResult r => [String] -> IO r
+cardanoWalletLauncherCLI = command [] "cardano-wallet-launcher"
+
+cardanoWalletCLI :: CmdResult r => [String] -> IO r
+cardanoWalletCLI = command [] "cardano-wallet"
+
+generateMnemonicsViaCLI :: CmdResult r => [String] -> IO r
+generateMnemonicsViaCLI args = cardanoWalletCLI (["mnemonic", "generate"] ++ args)
+
+createWalletViaCLI :: [String] -> String -> String -> String -> IO ExitCode
+createWalletViaCLI args mnemonics mnemonics2 passphrase = do
+    let fullArgs = ["wallet", "create", "--port", "1337"] ++ args
+    let process = (proc "cardano-wallet" fullArgs)
+            { std_in = CreatePipe, std_out = CreatePipe, std_err = CreatePipe }
+    withCreateProcess process $ \stdIn _ _ h -> do
+        case stdIn of
+            Nothing -> return ()
```

Comment:
I wonder if this should "throw". Since we use `CreatePipe` this _must_ be `Just`. If not, the test might hang forever.